packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/feature_login/ui_components/login_form.dart

```diff
@@ -73,9 +72,13 @@ class _LoginFormState extends State<LoginForm> {
           ),
           SizedBox(height: context.designSystem.spacing.l),
           GestureDetector(
-            onTap: () {
-              // Implement your logic here
-            },
+            onTap: () => showBlurredBottomSheet(
+              context: context,
+              builder: (BuildContext context) => MessagePanelWidget(
+                message: context.l10n.featureLogin.notImplementedMessage,
+                messageState: MessagePanelState.lessImportant,
+              ),
+            ),
             child: Text(context.l10n.featureLogin.forgottenPassword),
```

Comment:
Can we use the text button here ?

Reply:
Using TextButton, it will have a bit different styling. Do you think it will fitt better? - The first one is TextButton, but the second use the old implementation (in light and dark mode)
![Screenshot_20250108_151726](https://github.com/user-attachments/assets/a74ab67a-e37c-4df8-bd30-e93bbb9cc93b)
![Screenshot_20250108_151747](https://github.com/user-attachments/assets/b410c057-80a3-4f22-afa8-c00b3ee067d0)

Reply:
I think it's ok

Reply:
Updаted